packages/reports/src/reports/components/ApiInput.jsx

```diff
@@ -27,19 +34,59 @@ function ApiInput({defaultValue, id, onChange, onEnterPress}) {
     onChange(e.target.value);
   };
 
+  const block = useBlock(id);
+
+  /**
+   * The block content data to use for rendering.
+   * If no "blockStateContent" prop is given, defaults to the redux store value of block.
+   * Use the "blockStateContent" prop to provide override values if you want to show a
+   * live preview of unsaved changes.
+   */
+  const blockContent = blockStateContent || getBlockContent(block, locale);
+
+  const {duration} = useMemo(() => {
+
+    /** @type {import("./types/PreviewAdapters").BlockPreviewAdapterParams} */
+    const adapterParams = {active: true, block, blockContent, debug: true, locale, variables};
```

Comment:
`duration` is only used by generators, as is `ApiInput`. The generatorPreviewAdapter should be used directly here (not using the lookup). It only has two arguments, `block` and `active`, so you shouldn't need all these arguments.

(separately, education-wise, your use of `locale` and `variables` in this adapterParams line would not necessarily be up to date, because they are not listed in the `[variables]` in the dependency argument)

Because you only need `block` and `active` here, you can remove the `blockStateContent` and `locale` that you added. Just directly use the generatorAdapter directly, the `block` from `useBlock`, and `active:true`